packages/trpc/server/routers/viewer/eventTypes/update.handler.ts

```diff
@@ -518,6 +527,36 @@ export const updateHandler = async ({ ctx, input }: UpdateOptions) => {
     }
   }
 
+  if (calVideoSettings) {
+    await ctx.prisma.calVideoSettings.upsert({
+      where: { eventTypeId: id },
+      update: {
+        disableRecordingForGuests: calVideoSettings.disableRecordingForGuests ?? false,
+        disableRecordingForOrganizer: calVideoSettings.disableRecordingForOrganizer ?? false,
+        redirectUrlOnExit: calVideoSettings.redirectUrlOnExit ?? null,
+        updatedAt: new Date(),
+      },
+      create: {
+        disableRecordingForGuests: calVideoSettings.disableRecordingForGuests ?? false,
+        disableRecordingForOrganizer: calVideoSettings.disableRecordingForOrganizer ?? false,
+        redirectUrlOnExit: calVideoSettings.redirectUrlOnExit ?? null,
+        eventTypeId: id,
+      },
+    });
+  }
+
+  const isCalVideoLocationActive = locations
+    ? locations.some((location) => location.type === "integrations:daily")
```

Comment:
We should define a constant for "integrations:daily" and use that across the app.